behandlingslager/domene/src/main/java/no/nav/foreldrepenger/behandlingslager/behandling/ytelsefordeling/periode/OppgittPeriodeEntitet.java

```diff
@@ -329,6 +329,10 @@ public boolean isVedtaksperiode() {
         return getPeriodeKilde().equals(FordelingPeriodeKilde.TIDLIGERE_VEDTAK);
     }
 
+    public boolean erAktivitetskravMedMorArbeid() {
```

Comment:
Det er denne omgangen. Men vi får kanskje også FORELDREPENGER. + ikkeMor.
Passer bedre der det brukes ?

Reply:
Ok for å sjekke på både periodetype og morsaktivitet
For BFHR får vi foreldrepenger - 10 uker uten mors aktivitet + opptil 30 med mors aktivitet